client/tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
```

Comment:
This is the newer jsx flag, we should keep it enabled (revert changes). You might need to upgrade your typescript version if you're getting errors.